src/ui_utils.c

```diff
@@ -2477,10 +2477,24 @@ void ui_init_builder(void)
 static void init_custom_style(void)
 {
 #if GTK_CHECK_VERSION(3, 0, 0)
-	gchar *css_file = g_build_filename(app->datadir, "geany.css", NULL);
+	const struct {
+		guint version;
+		const gchar *file;
+	} css_files[] = {
+		{ 20, "geany-3.20.css" },
+		{ 0, "geany-3.0.css" },
+	};
```

Comment:
Use [Allman style](https://en.wikipedia.org/wiki/Indent_style#Allman_style) [braces on their own line](https://github.com/geany/geany/blob/1.27.0/HACKING#L249) instead of [K&R style](https://en.wikipedia.org/wiki/Indent_style#K.26R_style) braces on the same line. That being said, one could argue you're just following [existing code that breaks the convention](https://github.com/geany/geany/blob/1.27.0/src/filetypes.c#L619) :)

In theory it would be like this:

```c
const struct 
{
	guint version;
	const gchar *file;
}
css_files[] =
{
	{ 20, "geany-3.20.css" },
	{ 0, "geany-3.0.css" },
};
```